packages/bpk-component-breadcrumb/src/BpkBreadcrumbItem-test.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import BpkBreadcrumbItem from './BpkBreadcrumbItem';
+
+describe('BpkBreadcrumbItem', () => {
+  it('should render correctly', () => {
+    const tree = renderer
+      .create(
+        <BpkBreadcrumbItem
+          href="https://backpack.github.io/"
+          aria-current="page"
+        >
+          Backpack
+        </BpkBreadcrumbItem>,
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render correctly with a "active" prop', () => {
+    const tree = renderer
+      .create(<BpkBreadcrumbItem active>Backpack</BpkBreadcrumbItem>)
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render correctly with a custom class name', () => {
+    const tree = renderer
+      .create(
+        <BpkBreadcrumbItem
+          href="https://backpack.github.io/"
+          className="my-custom-class"
+        >
+          Backpack
+        </BpkBreadcrumbItem>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render correctly with arbitrary props', () => {
+    const tree = renderer
+      .create(
+        <BpkBreadcrumbItem
+          href="https://backpack.github.io/"
+          testID="arbitrary value" // <-- arbitrary prop
+        >
+          Backpack
+        </BpkBreadcrumbItem>,
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render correctly with arbitrary props', () => {
+    const tree = renderer
+      .create(
+        <BpkBreadcrumbItem
+          href="https://backpack.github.io/"
+          linkProps={{ target: '_blank' }} // <-- arbitrary prop
```

Comment:
Should have a `blank` prop that sets both `target` and `rel` on `BpkBreadCrumbItem`? It doesn't seem reasonable for breadcrumb items to be external links?

Reply:
The component uses BPKLink and passes the linkprops to it. So this would be taken care of.

Reply:
> It doesn't seem reasonable for breadcrumb items to be external links?

It's unlikely to happen yeah - I felt that we should expose a mechanism for consumers to pass props to the inner `BpkLink`, and I think the `target` prop here is just being used as an example of an arbitrary prop. Maybe `testId` would be clearer to remove any confusion.